src/app/Http/Controllers/AccountController.php

```diff
@@ -348,4 +353,57 @@ public function updateMail(Request $request)
         }
         return redirect('/');
     }
+
+    public function pdfTicket(string $ticketId): Response {
+        $user = Auth::user();
+        $participant = EventParticipant::where('id', $ticketId)->first();
+        if ($user->id != $participant->user_id) {
+            return response()->view('errors.403', [], Response::HTTP_FORBIDDEN);
```

Comment:
This is not a great idea, since the user does not know what happened. If you want to keep the current way with returning a response directly to the errors page, plase add a ViewErrorBag. Something like:

```
  $viewErrorBag = (new ViewErrorBag)->put('default',
            (
                new MessageBag(
                    [
                        0 => [
                            'Your user is not allowed to get this ticket.'
                        ]
                    ]
                        ))
            );
        return response()->view('errors.403', ["errors" => $viewErrorBag], Response::HTTP_FORBIDDEN);
        
```

Reply:
Also without an ViewErrorBag the page will just return a error 500 in production, because $errors is not defined

Reply:
ofc, the error should be localized here

Reply:
Implemented your suggestions.